.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "m2e-core-tests"]
 	path = m2e-core-tests
-	url = https://github.com/tesla/m2e-core-tests.git
```

Comment:
This is only for testing the m2e-core test changes and has to be reverted before submission.

Reply:
Done.